pex/pip.py

```diff
@@ -521,6 +555,31 @@ def spawn_install_wheel(
         install_cmd.append(wheel)
         return self._spawn_pip_isolated_job(install_cmd, cache=cache, interpreter=interpreter)
 
+    def spawn_debug(
+        self,
+        platform,  # type: str
+        impl,  # type: str
+        version,  # type: str
+        abi,  # type: str
+        manylinux=None,  # type: Optional[str]
+    ):
+        # type: (...) -> Job
+        # N.B.: Pip gives fair warning:
```

Comment:
Maybe worth adding something like this: "However, we can use it because we vendor a pinned version of pip".

Reply:
I'll pass on this since it's redundant if you're reading this file. It's central to all aspects of calling pip that we have a pinned version - vendored or not - so we can rely on flags for all other three commands as well.